mindmeld/bot/webex_bot_server.py

```diff
@@ -60,7 +61,7 @@ def __init__(self, name, app_path, nlp=None, webhook_id=None, access_token=None)
         self.access_token_with_bearer = ACCESS_TOKEN_WITH_BEARER + self.access_token
 
         @self.app.route('/', methods=['POST'])
-        def handle_message():
+        def handle_message():  # pylint: disable=unused-variable
```

Comment:
If this function is not being used, we should delete it

Reply:
It is the main function to handle message